sanitizers/src/test/java/com/example/BUILD.bazel

```
@@ -58,6 +58,19 @@ java_fuzz_target_test(
     ],
 )
 
+java_fuzz_target_test(
+    name = "FilePathTraversal",
+    srcs = [
+        "FilePathTraversal.java",
+    ],
+    allowed_findings = [
+        "com.code_intelligence.jazzer.api.FuzzerSecurityIssueCritical",
+    ],
+    target_class = "com.example.FilePathTraversal",
+    #not clear why reproducer doesn't work TODO -- fix this
+    verify_crash_reproducer = False,
+)
+
```

- added deps for using Jazzer annotations in the example

```suggestion
java_fuzz_target_test(
    name = "FilePathTraversal",
    srcs = [
        "FilePathTraversal.java",
    ],
    allowed_findings = [
        "com.code_intelligence.jazzer.api.FuzzerSecurityIssueCritical",
    ],
    target_class = "com.example.FilePathTraversal",
    #not clear why reproducer doesn't work TODO -- fix this
    verify_crash_reproducer = False,
    deps = [
        "//src/main/java/com/code_intelligence/jazzer/mutation/annotation",
    ],
)
)

```